misk/src/main/kotlin/misk/moshi/BigDecimalAdapter.kt

```diff
@@ -0,0 +1,13 @@
+package misk.moshi
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import java.math.BigDecimal
+
+object BigDecimalAdapter {
```

Comment:
not sure if we should create a `adapters` package or if this is good for the time been.

Reply:
Let's create the package and put this adapter there but skip on consolidating the other adapters into it since it will be a main to migrate all of the imports.

Reply:
this is done.